bluemira/equilibria/coils/_grouping.py

```diff
@@ -1211,6 +1220,11 @@ def remove_coil(self, *coil_name: str, _top_level: bool = True) -> list[str] | N
         self.control = list(set(self.control) & set(self.name))
         return removed_coils
 
+    @property
+    def n_control(self) -> float:
+        """Number of coils being actively controlled"""
+        return len(self._control)
```

Comment:
Possibly use the property?

```suggestion
        return len(self.control)
```